### PR TITLE
Propose secure chaincode execution for Fabric

### PR DESCRIPTION
**Pull request by mbrandenburger**
Signed-off-by: Marcus Brandenburger <bur@zurich.ibm.com>

### CODE REVIEW

**Comment by trbs** [comment]
LGTM

Could there be some overlap with project (proposals) like https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/13 ?

**Comment by lehors** [comment]
@trbs That's what I was wondering in my comment yesterday on #13. However, it is said to be a complete platform while this is specific to Fabric and only touches on the execution of the chaincode. So, while they fall in the same space I think they are significantly different and I'm not sure there is going to be much overlap.
It's worth having the two labs know about each other though.

LGTM

**Comment by mbrandenburger** [comment]
@trbs I agree with Arnaud, our project focuses on enabling secure chaincode execution for Fabric using Intel SGX. We presented our work at the last Hackfest in Amsterdam and got many positive response already from the community.

File: labs/fabric-secure-chaincode.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+registration and transaction verification are provided.
+
+# Scope of Lab
+This lab proposes an architecture to enable Secure Chaincode Execution
+using Intel SGX for Hyperledger Fabric.  We provide an initial
+proof-of-concept implementation of the proposed architecture. The main goal of
+this lab is to discuss and refine the proposed architecture involving
+the Hyperledger community.
+
+# Initial Committers
```
Comment (tkuhrt) [inline labs/fabric-secure-chaincode.md]: Can you add the Github ID's for each of the initial committers? Gari too if he wants to have commit rights.

**Comment by tkuhrt** [comment]
Added a single comment to the proposal to address before we move forward and create this lab.

File: labs/fabric-secure-chaincode.md (hunk 1 above)
Comment (mbrandenburger) [reply labs/fabric-secure-chaincode.md]: Thanks! Added!

File: labs/fabric-secure-chaincode.md [hunk 2]
```diff
@@ -30,10 +30,10 @@ this lab is to discuss and refine the proposed architecture involving
 the Hyperledger community.
 
 # Initial Committers
-- Marcus Brandenburger (bur@zurich.ibm.com)
-- Christian Cachin (cca@zurich.ibm.com)
+- https://github.com/mbrandenburger Marcus Brandenburger (bur@zurich.ibm.com)
+- https://github.com/cca88 Christian Cachin (cca@zurich.ibm.com)
 - Rüdiger Kapitza (kapitza@ibr.cs.tu-bs.de)
```
Comment (tkuhrt) [inline labs/fabric-secure-chaincode.md]: Does Rüdiger not have a Github ID?

File: labs/fabric-secure-chaincode.md [hunk 3]
```diff
@@ -30,10 +30,10 @@ this lab is to discuss and refine the proposed architecture involving
...
-- Alessandro Sorniotti (aso@zurich.ibm.com)
+- https://github.com/ale-linux Alessandro Sorniotti (aso@zurich.ibm.com)
 
 # Sponsor
-Gari Sighn (garis@us.ibm.com)
+https://github.com/mastersingh24 Gari Sighn (garis@us.ibm.com)
```
Comment (tkuhrt) [inline labs/fabric-secure-chaincode.md]: Typo in Gari's last name.

**Comment by mbrandenburger** [comment]
@tkuhrt Typos fixed and list of Initial Committers completed. Thank you

**Comment by tkuhrt** [comment]
New lab created: https://github.com/hyperledger-labs/fabric-secure-chaincode

**Pull request merged**